.github/workflows/ci_tests.yaml

```
@@ -185,4 +225,4 @@ jobs:
         test/externaldata/dummy-provider/scripts/generate-tls-certificate.sh
         docker build -t dummy-provider:test -f test/externaldata/dummy-provider/Dockerfile test/externaldata/dummy-provider
         kind load docker-image --name kind dummy-provider:test
-        make test-e2e GATEKEEPER_NAMESPACE=${{ matrix.NAMESPACE }} ENABLE_MUTATION_TESTS=1
+        make test-e2e GATEKEEPER_NAMESPACE=${{ matrix.NAMESPACE }} ENABLE_MUTATION_TESTS=1
```

nit: Missing trailing newline
```suggestion
        make test-e2e GATEKEEPER_NAMESPACE=${{ matrix.NAMESPACE }} ENABLE_MUTATION_TESTS=1

```